model_compression_toolkit/gptq/keras/quantization_facade.py

```diff
@@ -94,19 +96,37 @@ def get_keras_gptq_config(n_epochs: int,
             >>> import model_compression_toolkit as mct
             >>> import tensorflow as tf
 
-            Create a GradientPTQConfigV2 to run for 5 epochs:
+            Create a GradientPTQConfig to run for 5 epochs:
 
             >>> gptq_conf = mct.gptq.get_keras_gptq_config(n_epochs=5)
 
             Other Tensorflow optimizers can be passed:
 
             >>> gptq_conf = mct.gptq.get_keras_gptq_config(n_epochs=3, optimizer=tf.keras.optimizers.Nadam())
 
-            The configuration can be passed to :func:`~model_compression_toolkit.keras_post_training_quantization` in order to quantize a keras model using gptq.
+            To enable Gradual Activation Quantization with non-default settings build GradualActivationQuantizationConfig:
+
+            >>> gradual_act_conf = mct.gptq.GradualActivationQuantizationConfig(mct.gptq.QFractionLinearAnnealingConfig(initial_q_fraction=0.2))
```

Comment:
this looks very scary, do we really need to specify `initial_q_fraction`?
 even providing the `GradualActivationQuantizationConfig` and not use a default one seems like to much understanding of the algorithm to ask for from the user

Reply:
It's possible to remove this, but we need to demonstrate to the user how to use and configure GradualActivationQuantizationConfig somehow.

Do you think this should be in the GPTQ tutorial instead? Even though I think it should be here since this is what will be generated in the docs pages.

Reply:
I  do think it should be in a tutorial, and here we should only have a "plug-and-play" example. I think that just giving a line of code that configures something without getting into the meaning of every argument and module that is used can be confusing